associations/models/page.py

```diff
@@ -1,12 +1,30 @@
 from django.db import models
 
-from associations.models.association import Association
+from associations.models import Association
+from authentication.models import User
 
 
 class Page(models.Model):
     id = models.AutoField(primary_key=True)
 
     association = models.ForeignKey(Association, on_delete=models.CASCADE, related_name="pages")
+    authors = models.ManyToManyField(User)
+
+    creation_date = models.DateTimeField(auto_now_add=True)
+    last_update_date = models.DateTimeField(auto_now=True)
 
     title = models.CharField(max_length=200)
     text = models.TextField(blank=True, default='')
+
+    NEWS = 'NEWS'
+    """A Page which will be displayed in the news feed of the association."""
+
+    STATIC = 'STATIC'
+    """A Page which can be accessed from the association main page."""
+
+    PAGE_TYPES = (
+        (NEWS, 'News page'),
+        (STATIC, 'Static page')
```

Comment:
Maybe not useful to have a "display name" since it will be handled in the front-end. Might as well define 
```
PAGE_TYPES = {
     "NEWS" = "NEWS",
     "STATIC" = "STATIC"
}
```

Reply:
Well, they might be useful in the admin pane at least. You never know, and I thought it does no harm to keep them here.